OneClick/src/main/java/net/runelite/client/plugins/oneclick/AlchItem.java

```diff
@@ -0,0 +1,19 @@
+package net.runelite.client.plugins.oneclick;
+
+import lombok.Getter;
+
+class AlchItem
```

Comment:
Getter annotation on the class

Reply:
Wtf IntelliJ I literally already had that https://i.imgur.com/GjxSjo0.png